### DISCUSSION TITLE
想问下这个是正常的吗）

### DISCUSSION THREAD

**Discussion by Kite95** [mikumifa/biliTickerBuy#389] [提问专区]
![234](https://github.com/mikumifa/biliTickerBuy/assets/99546365/2cdebff0-3e54-4381-be87-6d37686c3b8e)

**Comment by Kite95** [comment]
没事）解决了

**Comment by littlemayi** [comment]
哥，怎么解决的

**Reply by Kite95** [reply]
等个十几分钟，实在不行重启路由器

**Reply by littlemayi** [reply]
我昨天晚上还有今天起来就挂着，重启和改间隔秒都还是跳这个
